README.mdx

```diff
@@ -0,0 +1,206 @@
+# TODO: Expand on outline
+
+> :warning: This guidance is in prep for a Playground tutorial, but attachments must first be enabled on Playground for
+this guidance to make sense.
+
+# KittyVerse + KittyHats
+
```

Comment:
Do you think it would be good to add the reference of kittyVerse & kittyHats repo ? or a small context about them ?